pb/terrarium/provider/provider.proto

```diff
@@ -8,6 +8,12 @@ service ProviderPublisher {
   rpc EndProvider(EndProviderRequest) returns (Response) {}
 }
 
+service ProviderConsumer {
```

Comment:
We don't need this service only the internal ones as the ability to download the various files is via a rest API like for modules. The Rest API handler will directly use the internal service.